phenotrex/cli/generic_func.py

```diff
@@ -83,13 +83,17 @@ def generic_cv(type, genotype, phenotype, folds, replicates, threads, verb, opti
     reduce_features = True if n_features is not None else False
     use_groups = groups is not None
     logger.info(f'Running CV...')
-    score_mean, score_sd, misclass = clf.crossvalidate(records=training_records, cv=folds,
+    #TODO change return fields
+    score_mean_sd, misclass = clf.crossvalidate(records=training_records, cv=folds,
                                                        n_replicates=replicates, groups=use_groups,
                                                        n_jobs=threads,
                                                        reduce_features=reduce_features,
                                                        n_features=n_features,
                                                        demote=not verb)
```

Comment:
Please check indentation here. Perhaps we can also have one argument per line:
```suggestion
score_mean_sd, misclass = clf.crossvalidate(
    records=training_records,
    cv=folds,
    n_replicates=replicates,
    groups=use_groups,
    n_jobs=threads,
    reduce_features=reduce_features,
    n_features=n_features,
    demote=not verb,
)
```

Reply:
Right, thanks